server/routes/fileMulter.js

```diff
@@ -21,7 +21,31 @@ const upload = multer({ storage: fileStorageEngine })
 
 router.post('/single', VerifyToken, upload.single('file'), async (req, res) => {
 
-  console.log(req.query)
+  // validating image types in case someone attempts to access route without using frontend
+  let validImageTypes = ["png", "PNG", "jpeg", "jpg"]
+
+  // Checking to see if the extension is one of the valid types
+  const imageTypePath = req.query.imageName.split('.')
+  const imageType = imageTypePath[imageTypePath.length - 1]
+  const validInput = validImageTypes.includes(imageType);
+
+  // If it isn't, return and allow user to input valid image
+  if (!validInput) {
+    console.log('Invalid file type. Please upload an image with the extension .jpg or .png')
```

Comment:
is this debugging? also if not, you are only saying 2 of the 4 file types but since this is the server, this should not print anything really

Reply:
It's disallowing any wrong file type. I'll send a message back instead, good idea.